README.md

```diff
@@ -159,6 +169,48 @@ directory to export a version-specific property:
 | tests.relevant   | List of relevant tests classes (a test class is relevant if, when executed, the JVM loads at least one of the modified classes) |
 | tests.trigger    | List of test methods that trigger (expose) the bug                                  |
 
+Export project-specific metadata
+--------------------------------
+Use `defects4j query -p <pid> -q <field_list> [-o <output_file>] [-D|-A]` 
+to generate a CSV file containing a set of requested metadata for each bug 
+in a specific project.
+
+By default, `defects4j query` returns a list of active bug IDs for a project.
+To request specific metadata, the `-q` flag should be provided with a 
+comma-separated list of variables from the list below. For example, 
+`defects4j query -p Chart -q "report.id,report.url"` will provide the a list of
+all active bug IDs, along with the bug report ID and bug report URL for each.
+
+
+| Property              | Description                                                                         |
+|-----------------------|-------------------------------------------------------------------------------------|
+| bug.id                | Assigned bug IDs (included in all results)                                          |
+| project.id            | Assigned project ID                                                                 |
+| project.name          | Original repository name                                                            |
+| project.build.file    | Location of the Defects4J build file for the project                                |
+| project.vcs           | Version control system used by the project                                          |
+| project.repository    | Location of the project repository                                                  |
+| project.commit.db     | Location of the CSV containing information on that bug                              |
```

Comment:
Should this be the field for the active-bugs.csv? Are there now two fields for active and deprecated bugs?

Reply:
Changed to project.bugs.csv in latest commit